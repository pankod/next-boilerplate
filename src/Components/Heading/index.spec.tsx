// #region Global Imports
import * as React from "react";
import { shallow, mount } from "enzyme";
// #endregion Global Imports

// #region Local Imports
import { Heading } from "@Components";
// #endregion Local Imports

describe("Components", () => {
    describe("Heading", () => {
        it("should render without throwing an error", () => {
            const wrap = shallow(<Heading text="World" />);

            expect(wrap.find("div.title").exists()).toBe(true);
        });

        it("should match snapshot", () => {
            const wrapper = mount(<Heading text="Test" />);
            expect(wrapper).toMatchSnapshot();
        });
    });
});
