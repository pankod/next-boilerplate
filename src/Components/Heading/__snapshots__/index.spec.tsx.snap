// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Heading should match snapshot 1`] = `
<Heading
  text="Test"
>
  <div
    className="title"
  >
    <span
      className="title__back"
    >
      Test
    </span>
    <span
      className="title__front"
    >
      Test
    </span>
  </div>
</Heading>
`;
